synapse/metrics/_exposition.py

```diff
@@ -118,12 +116,12 @@ def generate_latest(registry, emit_help=False):
         # Output in the old format for compatibility.
         if emit_help:
             output.append(
-                "# HELP {0} {1}\n".format(
+                "# HELP {} {}\n".format(
                     mname,
                     metric.documentation.replace("\\", r"\\").replace("\n", r"\n"),
```

Comment:
a note; `pyupgrade` doesn't inline to f-strings if the arguments to the string formatting function are sufficiently complex, so it encountered this and said "nope", thus only the removal of positional arguments, because they're redundant.